dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/dataset/metadata/inputRenderer/suggestion/GrantSuggestionHandlerTest.java

```diff
@@ -0,0 +1,48 @@
+package edu.harvard.iq.dataverse.dataset.metadata.inputRenderer.suggestion;
+
+import com.google.common.collect.Lists;
+import io.vavr.API;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import java.util.HashMap;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class GrantSuggestionHandlerTest {
+
+
+    GrantSuggestionDao grantSuggestionDao = Mockito.mock(GrantSuggestionDao.class);
+
+    // -------------------- TESTS --------------------
+
+    @Test
```

Comment:
Not sure, but maybe an additional test where you pass not-empty map that resolves to empty filters would help.

Reply:
I'm guessing this is now obsolete since because of our previous conversation.